README.adoc

```diff
@@ -44,16 +45,43 @@ image:images/net_ok.png[,250]
 image:images/boot_screen.png[,250]
 image:images/VM_running.png[,250]
 
-. It can be good idea at this point to setup an SSH connection
-.. J'ai oublie comment on fait
+. It can be good idea, at this point, to setup an SSH connection :
+.. Find the ip address of the machine. You can use the following command :
++
+[,bash]
+----
+ip a
+----
++
+The `_a_` stands for _address_. This will display information about all network interfaces, your ip address is somewhere here.
+.. Set your root user password with :
++
+[,bash]
+----
+passwd
+----
++
+the default password in the VM will be _1234_
```

Comment:
the default password information can be say later.
When you explain how use the VM after the Linux installation